tests/packages/vendored-meson/meson_options.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2023 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+option('custom-cli-used', type: 'boolean', value: false)
```

Comment:
I would call the option `custom-meson-used` to reflect the renamed configuration entry, but it is ok like this too.